opm/simulators/aquifers/AquiferCarterTracy.hpp

```diff
@@ -145,6 +147,9 @@ namespace Opm
                 }
             }
 
+            void assignRestartData(const data::AquiferData& /* xaq */) override
```

Comment:
I think we have not finished the restarting implementation for AquiferCarterTracy.  Maybe it is good to throw a message here.

Reply:
> think we have not finished the restarting implementation for AquiferCarterTracy

That is correct.

> Maybe it is good to throw a message here.

Okay.  I'll update accordingly.